<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="26" total="26" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="416706" started-at="2017-02-07T13:27:00Z" finished-at="2017-02-07T13:33:56Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="416706" started-at="2017-02-07T13:27:00Z" finished-at="2017-02-07T13:33:56Z">
      <class name="com.elm.mosa.portal.test.automation.Configure_Execute">
        <test-method status="PASS" signature="setup()[pri:0, instance:com.elm.mosa.portal.test.automation.Configure_Execute@91151ba]" name="setup" is-config="true" duration-ms="9407" started-at="2017-02-07T16:27:00Z" finished-at="2017-02-07T16:27:09Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="FAIL" signature="ConfigureAE()[pri:1, instance:com.elm.mosa.portal.test.automation.Configure_Execute@91151ba]" name="ConfigureAE" duration-ms="9624" started-at="2017-02-07T16:27:09Z" finished-at="2017-02-07T16:27:19Z">
          <exception class="org.junit.ComparisonFailure">
            <message>
              <![CDATA[expected:<[wrong]> but was:<[item.configured === true]>]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.junit.ComparisonFailure: expected:<[wrong]> but was:<[item.configured === true]>
	at org.junit.Assert.assertEquals(Assert.java:115)
	at org.junit.Assert.assertEquals(Assert.java:144)
	at com.elm.mosa.portal.test.automation.Configuration.ConfigureAE(Configuration.java:72)
	at com.elm.mosa.portal.test.automation.Configure_Execute.ConfigureAE(Configure_Execute.java:54)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1129)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:746)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1264)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1189)
	at org.testng.TestNG.runSuites(TestNG.java:1104)
	at org.testng.TestNG.run(TestNG.java:1076)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:152)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:57)
]]>
            </full-stacktrace>
          </exception> <!-- org.junit.ComparisonFailure -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ConfigureAE -->
        <test-method status="PASS" signature="End()[pri:0, instance:com.elm.mosa.portal.test.automation.Configure_Execute@91151ba]" name="End" is-config="true" duration-ms="1119" started-at="2017-02-07T16:27:19Z" finished-at="2017-02-07T16:27:20Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- End -->
        <test-method status="PASS" signature="setup()[pri:0, instance:com.elm.mosa.portal.test.automation.Configure_Execute@91151ba]" name="setup" is-config="true" duration-ms="5329" started-at="2017-02-07T16:27:20Z" finished-at="2017-02-07T16:27:25Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="FAIL" signature="ConfigureHE()[pri:2, instance:com.elm.mosa.portal.test.automation.Configure_Execute@91151ba]" name="ConfigureHE" duration-ms="13087" started-at="2017-02-07T16:27:25Z" finished-at="2017-02-07T16:27:38Z">
          <exception class="org.junit.ComparisonFailure">
            <message>
              <![CDATA[expected:<[wrong]> but was:<[item.configured === true]>]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.junit.ComparisonFailure: expected:<[wrong]> but was:<[item.configured === true]>
	at org.junit.Assert.assertEquals(Assert.java:115)
	at org.junit.Assert.assertEquals(Assert.java:144)
	at com.elm.mosa.portal.test.automation.Configuration.ConfigureHE(Configuration.java:82)
	at com.elm.mosa.portal.test.automation.Configure_Execute.ConfigureHE(Configure_Execute.java:60)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1129)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:746)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1264)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1189)
	at org.testng.TestNG.runSuites(TestNG.java:1104)
	at org.testng.TestNG.run(TestNG.java:1076)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:152)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:57)
]]>
            </full-stacktrace>
          </exception> <!-- org.junit.ComparisonFailure -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ConfigureHE -->
        <test-method status="PASS" signature="End()[pri:0, instance:com.elm.mosa.portal.test.automation.Configure_Execute@91151ba]" name="End" is-config="true" duration-ms="777" started-at="2017-02-07T16:27:38Z" finished-at="2017-02-07T16:27:39Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- End -->
        <test-method status="PASS" signature="setup()[pri:0, instance:com.elm.mosa.portal.test.automation.Configure_Execute@91151ba]" name="setup" is-config="true" duration-ms="4444" started-at="2017-02-07T16:27:39Z" finished-at="2017-02-07T16:27:44Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="FAIL" signature="ConfigureAI()[pri:3, instance:com.elm.mosa.portal.test.automation.Configure_Execute@91151ba]" name="ConfigureAI" duration-ms="12134" started-at="2017-02-07T16:27:44Z" finished-at="2017-02-07T16:27:56Z">
          <exception class="org.junit.ComparisonFailure">
            <message>
              <![CDATA[expected:<[wrong]> but was:<[item.configured === true]>]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.junit.ComparisonFailure: expected:<[wrong]> but was:<[item.configured === true]>
	at org.junit.Assert.assertEquals(Assert.java:115)
	at org.junit.Assert.assertEquals(Assert.java:144)
	at com.elm.mosa.portal.test.automation.Configuration.ConfigureAI(Configuration.java:90)
	at com.elm.mosa.portal.test.automation.Configure_Execute.ConfigureAI(Configure_Execute.java:66)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1129)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:746)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1264)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1189)
	at org.testng.TestNG.runSuites(TestNG.java:1104)
	at org.testng.TestNG.run(TestNG.java:1076)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:152)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:57)
]]>
            </full-stacktrace>
          </exception> <!-- org.junit.ComparisonFailure -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ConfigureAI -->
        <test-method status="PASS" signature="End()[pri:0, instance:com.elm.mosa.portal.test.automation.Configure_Execute@91151ba]" name="End" is-config="true" duration-ms="780" started-at="2017-02-07T16:27:56Z" finished-at="2017-02-07T16:27:56Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- End -->
        <test-method status="PASS" signature="setup()[pri:0, instance:com.elm.mosa.portal.test.automation.Configure_Execute@91151ba]" name="setup" is-config="true" duration-ms="4266" started-at="2017-02-07T16:27:56Z" finished-at="2017-02-07T16:28:01Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="FAIL" signature="ConfigureDQD()[pri:4, instance:com.elm.mosa.portal.test.automation.Configure_Execute@91151ba]" name="ConfigureDQD" duration-ms="11111" started-at="2017-02-07T16:28:01Z" finished-at="2017-02-07T16:28:12Z">
          <exception class="org.junit.ComparisonFailure">
            <message>
              <![CDATA[expected:<[wrong]> but was:<[item.configured === true]>]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.junit.ComparisonFailure: expected:<[wrong]> but was:<[item.configured === true]>
	at org.junit.Assert.assertEquals(Assert.java:115)
	at org.junit.Assert.assertEquals(Assert.java:144)
	at com.elm.mosa.portal.test.automation.Configuration.ConfigureDQdependent(Configuration.java:97)
	at com.elm.mosa.portal.test.automation.Configure_Execute.ConfigureDQD(Configure_Execute.java:72)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1129)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:746)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1264)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1189)
	at org.testng.TestNG.runSuites(TestNG.java:1104)
	at org.testng.TestNG.run(TestNG.java:1076)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:152)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:57)
]]>
            </full-stacktrace>
          </exception> <!-- org.junit.ComparisonFailure -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ConfigureDQD -->
        <test-method status="PASS" signature="End()[pri:0, instance:com.elm.mosa.portal.test.automation.Configure_Execute@91151ba]" name="End" is-config="true" duration-ms="781" started-at="2017-02-07T16:28:12Z" finished-at="2017-02-07T16:28:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- End -->
        <test-method status="PASS" signature="setup()[pri:0, instance:com.elm.mosa.portal.test.automation.Configure_Execute@91151ba]" name="setup" is-config="true" duration-ms="4519" started-at="2017-02-07T16:28:13Z" finished-at="2017-02-07T16:28:17Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="FAIL" signature="ConfigureDCD()[pri:5, instance:com.elm.mosa.portal.test.automation.Configure_Execute@91151ba]" name="ConfigureDCD" duration-ms="11100" started-at="2017-02-07T16:28:17Z" finished-at="2017-02-07T16:28:28Z">
          <exception class="org.junit.ComparisonFailure">
            <message>
              <![CDATA[expected:<[wrong]> but was:<[item.configured === true]>]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.junit.ComparisonFailure: expected:<[wrong]> but was:<[item.configured === true]>
	at org.junit.Assert.assertEquals(Assert.java:115)
	at org.junit.Assert.assertEquals(Assert.java:144)
	at com.elm.mosa.portal.test.automation.Configuration.ConfigureDCdependent(Configuration.java:105)
	at com.elm.mosa.portal.test.automation.Configure_Execute.ConfigureDCD(Configure_Execute.java:77)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1129)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:746)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1264)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1189)
	at org.testng.TestNG.runSuites(TestNG.java:1104)
	at org.testng.TestNG.run(TestNG.java:1076)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:152)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:57)
]]>
            </full-stacktrace>
          </exception> <!-- org.junit.ComparisonFailure -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ConfigureDCD -->
        <test-method status="PASS" signature="End()[pri:0, instance:com.elm.mosa.portal.test.automation.Configure_Execute@91151ba]" name="End" is-config="true" duration-ms="760" started-at="2017-02-07T16:28:28Z" finished-at="2017-02-07T16:28:29Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- End -->
        <test-method status="PASS" signature="setup()[pri:0, instance:com.elm.mosa.portal.test.automation.Configure_Execute@91151ba]" name="setup" is-config="true" duration-ms="4772" started-at="2017-02-07T16:28:29Z" finished-at="2017-02-07T16:28:34Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="FAIL" signature="ConfigureEWSD()[pri:6, instance:com.elm.mosa.portal.test.automation.Configure_Execute@91151ba]" name="ConfigureEWSD" duration-ms="11125" started-at="2017-02-07T16:28:34Z" finished-at="2017-02-07T16:28:45Z">
          <exception class="org.junit.ComparisonFailure">
            <message>
              <![CDATA[expected:<[wrong]> but was:<[item.configured === true]>]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.junit.ComparisonFailure: expected:<[wrong]> but was:<[item.configured === true]>
	at org.junit.Assert.assertEquals(Assert.java:115)
	at org.junit.Assert.assertEquals(Assert.java:144)
	at com.elm.mosa.portal.test.automation.Configuration.ConfigureEWSdepedent(Configuration.java:112)
	at com.elm.mosa.portal.test.automation.Configure_Execute.ConfigureEWSD(Configure_Execute.java:82)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1129)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:746)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1264)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1189)
	at org.testng.TestNG.runSuites(TestNG.java:1104)
	at org.testng.TestNG.run(TestNG.java:1076)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:152)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:57)
]]>
            </full-stacktrace>
          </exception> <!-- org.junit.ComparisonFailure -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ConfigureEWSD -->
        <test-method status="PASS" signature="End()[pri:0, instance:com.elm.mosa.portal.test.automation.Configure_Execute@91151ba]" name="End" is-config="true" duration-ms="764" started-at="2017-02-07T16:28:45Z" finished-at="2017-02-07T16:28:46Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- End -->
        <test-method status="PASS" signature="setup()[pri:0, instance:com.elm.mosa.portal.test.automation.Configure_Execute@91151ba]" name="setup" is-config="true" duration-ms="4177" started-at="2017-02-07T16:28:46Z" finished-at="2017-02-07T16:28:50Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="FAIL" signature="ConfigureCESD()[pri:7, instance:com.elm.mosa.portal.test.automation.Configure_Execute@91151ba]" name="ConfigureCESD" duration-ms="11152" started-at="2017-02-07T16:28:50Z" finished-at="2017-02-07T16:29:01Z">
          <exception class="org.junit.ComparisonFailure">
            <message>
              <![CDATA[expected:<[wrong]> but was:<[item.configured === true]>]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.junit.ComparisonFailure: expected:<[wrong]> but was:<[item.configured === true]>
	at org.junit.Assert.assertEquals(Assert.java:115)
	at org.junit.Assert.assertEquals(Assert.java:144)
	at com.elm.mosa.portal.test.automation.Configuration.ConfigureCESdependent(Configuration.java:119)
	at com.elm.mosa.portal.test.automation.Configure_Execute.ConfigureCESD(Configure_Execute.java:87)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1129)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:746)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1264)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1189)
	at org.testng.TestNG.runSuites(TestNG.java:1104)
	at org.testng.TestNG.run(TestNG.java:1076)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:152)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:57)
]]>
            </full-stacktrace>
          </exception> <!-- org.junit.ComparisonFailure -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ConfigureCESD -->
        <test-method status="PASS" signature="End()[pri:0, instance:com.elm.mosa.portal.test.automation.Configure_Execute@91151ba]" name="End" is-config="true" duration-ms="780" started-at="2017-02-07T16:29:01Z" finished-at="2017-02-07T16:29:02Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- End -->
        <test-method status="PASS" signature="setup()[pri:0, instance:com.elm.mosa.portal.test.automation.Configure_Execute@91151ba]" name="setup" is-config="true" duration-ms="3669" started-at="2017-02-07T16:29:02Z" finished-at="2017-02-07T16:29:05Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="FAIL" signature="ConfigureCED()[pri:8, instance:com.elm.mosa.portal.test.automation.Configure_Execute@91151ba]" name="ConfigureCED" duration-ms="11118" started-at="2017-02-07T16:29:05Z" finished-at="2017-02-07T16:29:17Z">
          <exception class="org.junit.ComparisonFailure">
            <message>
              <![CDATA[expected:<[wrong]> but was:<[item.configured === true]>]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.junit.ComparisonFailure: expected:<[wrong]> but was:<[item.configured === true]>
	at org.junit.Assert.assertEquals(Assert.java:115)
	at org.junit.Assert.assertEquals(Assert.java:144)
	at com.elm.mosa.portal.test.automation.Configuration.ConfigureCEdependent(Configuration.java:125)
	at com.elm.mosa.portal.test.automation.Configure_Execute.ConfigureCED(Configure_Execute.java:92)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1129)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:746)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1264)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1189)
	at org.testng.TestNG.runSuites(TestNG.java:1104)
	at org.testng.TestNG.run(TestNG.java:1076)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:152)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:57)
]]>
            </full-stacktrace>
          </exception> <!-- org.junit.ComparisonFailure -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ConfigureCED -->
        <test-method status="PASS" signature="End()[pri:0, instance:com.elm.mosa.portal.test.automation.Configure_Execute@91151ba]" name="End" is-config="true" duration-ms="779" started-at="2017-02-07T16:29:17Z" finished-at="2017-02-07T16:29:17Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- End -->
        <test-method status="PASS" signature="setup()[pri:0, instance:com.elm.mosa.portal.test.automation.Configure_Execute@91151ba]" name="setup" is-config="true" duration-ms="3813" started-at="2017-02-07T16:29:17Z" finished-at="2017-02-07T16:29:21Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="FAIL" signature="ConfigureDQ()[pri:9, instance:com.elm.mosa.portal.test.automation.Configure_Execute@91151ba]" name="ConfigureDQ" duration-ms="11150" started-at="2017-02-07T16:29:21Z" finished-at="2017-02-07T16:29:32Z">
          <exception class="org.junit.ComparisonFailure">
            <message>
              <![CDATA[expected:<[wrong]> but was:<[item.configured === true]>]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.junit.ComparisonFailure: expected:<[wrong]> but was:<[item.configured === true]>
	at org.junit.Assert.assertEquals(Assert.java:115)
	at org.junit.Assert.assertEquals(Assert.java:144)
	at com.elm.mosa.portal.test.automation.Configuration.configureDQ(Configuration.java:132)
	at com.elm.mosa.portal.test.automation.Configure_Execute.ConfigureDQ(Configure_Execute.java:97)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1129)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:746)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1264)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1189)
	at org.testng.TestNG.runSuites(TestNG.java:1104)
	at org.testng.TestNG.run(TestNG.java:1076)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:152)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:57)
]]>
            </full-stacktrace>
          </exception> <!-- org.junit.ComparisonFailure -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ConfigureDQ -->
        <test-method status="PASS" signature="End()[pri:0, instance:com.elm.mosa.portal.test.automation.Configure_Execute@91151ba]" name="End" is-config="true" duration-ms="770" started-at="2017-02-07T16:29:32Z" finished-at="2017-02-07T16:29:33Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- End -->
        <test-method status="PASS" signature="setup()[pri:0, instance:com.elm.mosa.portal.test.automation.Configure_Execute@91151ba]" name="setup" is-config="true" duration-ms="4095" started-at="2017-02-07T16:29:33Z" finished-at="2017-02-07T16:29:37Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="FAIL" signature="ConfigureDC()[pri:10, instance:com.elm.mosa.portal.test.automation.Configure_Execute@91151ba]" name="ConfigureDC" duration-ms="11151" started-at="2017-02-07T16:29:37Z" finished-at="2017-02-07T16:29:48Z">
          <exception class="org.junit.ComparisonFailure">
            <message>
              <![CDATA[expected:<[wrong]> but was:<[item.configured === true]>]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.junit.ComparisonFailure: expected:<[wrong]> but was:<[item.configured === true]>
	at org.junit.Assert.assertEquals(Assert.java:115)
	at org.junit.Assert.assertEquals(Assert.java:144)
	at com.elm.mosa.portal.test.automation.Configuration.configureDC(Configuration.java:139)
	at com.elm.mosa.portal.test.automation.Configure_Execute.ConfigureDC(Configure_Execute.java:102)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1129)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:746)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1264)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1189)
	at org.testng.TestNG.runSuites(TestNG.java:1104)
	at org.testng.TestNG.run(TestNG.java:1076)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:152)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:57)
]]>
            </full-stacktrace>
          </exception> <!-- org.junit.ComparisonFailure -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ConfigureDC -->
        <test-method status="PASS" signature="End()[pri:0, instance:com.elm.mosa.portal.test.automation.Configure_Execute@91151ba]" name="End" is-config="true" duration-ms="764" started-at="2017-02-07T16:29:48Z" finished-at="2017-02-07T16:29:49Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- End -->
        <test-method status="PASS" signature="setup()[pri:0, instance:com.elm.mosa.portal.test.automation.Configure_Execute@91151ba]" name="setup" is-config="true" duration-ms="4659" started-at="2017-02-07T16:29:49Z" finished-at="2017-02-07T16:29:54Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="FAIL" signature="ConfigureEWS()[pri:11, instance:com.elm.mosa.portal.test.automation.Configure_Execute@91151ba]" name="ConfigureEWS" duration-ms="11102" started-at="2017-02-07T16:29:54Z" finished-at="2017-02-07T16:30:05Z">
          <exception class="org.junit.ComparisonFailure">
            <message>
              <![CDATA[expected:<[wrong]> but was:<[item.configured === true]>]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.junit.ComparisonFailure: expected:<[wrong]> but was:<[item.configured === true]>
	at org.junit.Assert.assertEquals(Assert.java:115)
	at org.junit.Assert.assertEquals(Assert.java:144)
	at com.elm.mosa.portal.test.automation.Configuration.configureEWS(Configuration.java:145)
	at com.elm.mosa.portal.test.automation.Configure_Execute.ConfigureEWS(Configure_Execute.java:107)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1129)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:746)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1264)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1189)
	at org.testng.TestNG.runSuites(TestNG.java:1104)
	at org.testng.TestNG.run(TestNG.java:1076)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:152)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:57)
]]>
            </full-stacktrace>
          </exception> <!-- org.junit.ComparisonFailure -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ConfigureEWS -->
        <test-method status="PASS" signature="End()[pri:0, instance:com.elm.mosa.portal.test.automation.Configure_Execute@91151ba]" name="End" is-config="true" duration-ms="767" started-at="2017-02-07T16:30:05Z" finished-at="2017-02-07T16:30:06Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- End -->
        <test-method status="PASS" signature="setup()[pri:0, instance:com.elm.mosa.portal.test.automation.Configure_Execute@91151ba]" name="setup" is-config="true" duration-ms="3976" started-at="2017-02-07T16:30:06Z" finished-at="2017-02-07T16:30:10Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="FAIL" signature="ConfigureCESA()[pri:12, instance:com.elm.mosa.portal.test.automation.Configure_Execute@91151ba]" name="ConfigureCESA" duration-ms="11106" started-at="2017-02-07T16:30:10Z" finished-at="2017-02-07T16:30:21Z">
          <exception class="org.junit.ComparisonFailure">
            <message>
              <![CDATA[expected:<[wrong]> but was:<[item.configured === true]>]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.junit.ComparisonFailure: expected:<[wrong]> but was:<[item.configured === true]>
	at org.junit.Assert.assertEquals(Assert.java:115)
	at org.junit.Assert.assertEquals(Assert.java:144)
	at com.elm.mosa.portal.test.automation.Configuration.ConfigureCESA(Configuration.java:153)
	at com.elm.mosa.portal.test.automation.Configure_Execute.ConfigureCESA(Configure_Execute.java:112)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1129)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:746)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1264)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1189)
	at org.testng.TestNG.runSuites(TestNG.java:1104)
	at org.testng.TestNG.run(TestNG.java:1076)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:152)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:57)
]]>
            </full-stacktrace>
          </exception> <!-- org.junit.ComparisonFailure -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ConfigureCESA -->
        <test-method status="PASS" signature="End()[pri:0, instance:com.elm.mosa.portal.test.automation.Configure_Execute@91151ba]" name="End" is-config="true" duration-ms="789" started-at="2017-02-07T16:30:21Z" finished-at="2017-02-07T16:30:22Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- End -->
        <test-method status="PASS" signature="setup()[pri:0, instance:com.elm.mosa.portal.test.automation.Configure_Execute@91151ba]" name="setup" is-config="true" duration-ms="3587" started-at="2017-02-07T16:30:22Z" finished-at="2017-02-07T16:30:25Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="FAIL" signature="ConfigureCEA()[pri:13, instance:com.elm.mosa.portal.test.automation.Configure_Execute@91151ba]" name="ConfigureCEA" duration-ms="11127" started-at="2017-02-07T16:30:25Z" finished-at="2017-02-07T16:30:36Z">
          <exception class="org.junit.ComparisonFailure">
            <message>
              <![CDATA[expected:<[wrong]> but was:<[item.configured === true]>]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.junit.ComparisonFailure: expected:<[wrong]> but was:<[item.configured === true]>
	at org.junit.Assert.assertEquals(Assert.java:115)
	at org.junit.Assert.assertEquals(Assert.java:144)
	at com.elm.mosa.portal.test.automation.Configuration.ConfigureCEA(Configuration.java:159)
	at com.elm.mosa.portal.test.automation.Configure_Execute.ConfigureCEA(Configure_Execute.java:117)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1129)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:746)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1264)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1189)
	at org.testng.TestNG.runSuites(TestNG.java:1104)
	at org.testng.TestNG.run(TestNG.java:1076)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:152)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:57)
]]>
            </full-stacktrace>
          </exception> <!-- org.junit.ComparisonFailure -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ConfigureCEA -->
        <test-method status="PASS" signature="End()[pri:0, instance:com.elm.mosa.portal.test.automation.Configure_Execute@91151ba]" name="End" is-config="true" duration-ms="812" started-at="2017-02-07T16:30:36Z" finished-at="2017-02-07T16:30:37Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- End -->
        <test-method status="PASS" signature="setup()[pri:0, instance:com.elm.mosa.portal.test.automation.Configure_Execute@91151ba]" name="setup" is-config="true" duration-ms="3719" started-at="2017-02-07T16:30:37Z" finished-at="2017-02-07T16:30:41Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="FAIL" signature="ExecuteAE()[pri:14, instance:com.elm.mosa.portal.test.automation.Configure_Execute@91151ba]" name="ExecuteAE" duration-ms="10720" started-at="2017-02-07T16:30:41Z" finished-at="2017-02-07T16:30:52Z">
          <exception class="org.junit.ComparisonFailure">
            <message>
              <![CDATA[expected:<[wrong]> but was:<[item.executed === true]>]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.junit.ComparisonFailure: expected:<[wrong]> but was:<[item.executed === true]>
	at org.junit.Assert.assertEquals(Assert.java:115)
	at org.junit.Assert.assertEquals(Assert.java:144)
	at com.elm.mosa.portal.test.automation.Execution.Applicant_Extraction(Execution.java:167)
	at com.elm.mosa.portal.test.automation.Configure_Execute.ExecuteAE(Configure_Execute.java:122)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1129)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:746)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1264)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1189)
	at org.testng.TestNG.runSuites(TestNG.java:1104)
	at org.testng.TestNG.run(TestNG.java:1076)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:152)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:57)
]]>
            </full-stacktrace>
          </exception> <!-- org.junit.ComparisonFailure -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ExecuteAE -->
        <test-method status="PASS" signature="End()[pri:0, instance:com.elm.mosa.portal.test.automation.Configure_Execute@91151ba]" name="End" is-config="true" duration-ms="780" started-at="2017-02-07T16:30:52Z" finished-at="2017-02-07T16:30:52Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- End -->
        <test-method status="PASS" signature="setup()[pri:0, instance:com.elm.mosa.portal.test.automation.Configure_Execute@91151ba]" name="setup" is-config="true" duration-ms="3601" started-at="2017-02-07T16:30:52Z" finished-at="2017-02-07T16:30:56Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="FAIL" signature="ExecuteHE()[pri:15, instance:com.elm.mosa.portal.test.automation.Configure_Execute@91151ba]" name="ExecuteHE" duration-ms="10811" started-at="2017-02-07T16:30:56Z" finished-at="2017-02-07T16:31:07Z">
          <exception class="org.junit.ComparisonFailure">
            <message>
              <![CDATA[expected:<[Wrong]> but was:<[item.executed === true]>]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.junit.ComparisonFailure: expected:<[Wrong]> but was:<[item.executed === true]>
	at org.junit.Assert.assertEquals(Assert.java:115)
	at org.junit.Assert.assertEquals(Assert.java:144)
	at com.elm.mosa.portal.test.automation.Execution.Household_Extraction(Execution.java:175)
	at com.elm.mosa.portal.test.automation.Configure_Execute.ExecuteHE(Configure_Execute.java:127)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1129)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:746)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1264)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1189)
	at org.testng.TestNG.runSuites(TestNG.java:1104)
	at org.testng.TestNG.run(TestNG.java:1076)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:152)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:57)
]]>
            </full-stacktrace>
          </exception> <!-- org.junit.ComparisonFailure -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ExecuteHE -->
        <test-method status="PASS" signature="End()[pri:0, instance:com.elm.mosa.portal.test.automation.Configure_Execute@91151ba]" name="End" is-config="true" duration-ms="795" started-at="2017-02-07T16:31:07Z" finished-at="2017-02-07T16:31:08Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- End -->
        <test-method status="PASS" signature="setup()[pri:0, instance:com.elm.mosa.portal.test.automation.Configure_Execute@91151ba]" name="setup" is-config="true" duration-ms="3609" started-at="2017-02-07T16:31:08Z" finished-at="2017-02-07T16:31:11Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="FAIL" signature="ExecuteAI()[pri:16, instance:com.elm.mosa.portal.test.automation.Configure_Execute@91151ba]" name="ExecuteAI" duration-ms="10845" started-at="2017-02-07T16:31:11Z" finished-at="2017-02-07T16:31:22Z">
          <exception class="org.junit.ComparisonFailure">
            <message>
              <![CDATA[expected:<[Wrong]> but was:<[item.executed === true]>]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.junit.ComparisonFailure: expected:<[Wrong]> but was:<[item.executed === true]>
	at org.junit.Assert.assertEquals(Assert.java:115)
	at org.junit.Assert.assertEquals(Assert.java:144)
	at com.elm.mosa.portal.test.automation.Execution.Agency_Integration(Execution.java:183)
	at com.elm.mosa.portal.test.automation.Configure_Execute.ExecuteAI(Configure_Execute.java:133)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1129)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:746)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1264)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1189)
	at org.testng.TestNG.runSuites(TestNG.java:1104)
	at org.testng.TestNG.run(TestNG.java:1076)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:152)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:57)
]]>
            </full-stacktrace>
          </exception> <!-- org.junit.ComparisonFailure -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ExecuteAI -->
        <test-method status="PASS" signature="End()[pri:0, instance:com.elm.mosa.portal.test.automation.Configure_Execute@91151ba]" name="End" is-config="true" duration-ms="780" started-at="2017-02-07T16:31:22Z" finished-at="2017-02-07T16:31:23Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- End -->
        <test-method status="PASS" signature="setup()[pri:0, instance:com.elm.mosa.portal.test.automation.Configure_Execute@91151ba]" name="setup" is-config="true" duration-ms="4012" started-at="2017-02-07T16:31:23Z" finished-at="2017-02-07T16:31:27Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="FAIL" signature="ExecuteDQD()[pri:17, instance:com.elm.mosa.portal.test.automation.Configure_Execute@91151ba]" name="ExecuteDQD" duration-ms="10727" started-at="2017-02-07T16:31:27Z" finished-at="2017-02-07T16:31:37Z">
          <exception class="org.junit.ComparisonFailure">
            <message>
              <![CDATA[expected:<[Wrong]> but was:<[item.executed === true]>]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.junit.ComparisonFailure: expected:<[Wrong]> but was:<[item.executed === true]>
	at org.junit.Assert.assertEquals(Assert.java:115)
	at org.junit.Assert.assertEquals(Assert.java:144)
	at com.elm.mosa.portal.test.automation.Execution.Data_Quality_Dependent(Execution.java:189)
	at com.elm.mosa.portal.test.automation.Configure_Execute.ExecuteDQD(Configure_Execute.java:138)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1129)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:746)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1264)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1189)
	at org.testng.TestNG.runSuites(TestNG.java:1104)
	at org.testng.TestNG.run(TestNG.java:1076)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:152)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:57)
]]>
            </full-stacktrace>
          </exception> <!-- org.junit.ComparisonFailure -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ExecuteDQD -->
        <test-method status="PASS" signature="End()[pri:0, instance:com.elm.mosa.portal.test.automation.Configure_Execute@91151ba]" name="End" is-config="true" duration-ms="762" started-at="2017-02-07T16:31:37Z" finished-at="2017-02-07T16:31:38Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- End -->
        <test-method status="PASS" signature="setup()[pri:0, instance:com.elm.mosa.portal.test.automation.Configure_Execute@91151ba]" name="setup" is-config="true" duration-ms="4026" started-at="2017-02-07T16:31:38Z" finished-at="2017-02-07T16:31:42Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="FAIL" signature="ExecuteDCD()[pri:18, instance:com.elm.mosa.portal.test.automation.Configure_Execute@91151ba]" name="ExecuteDCD" duration-ms="10730" started-at="2017-02-07T16:31:42Z" finished-at="2017-02-07T16:31:53Z">
          <exception class="org.junit.ComparisonFailure">
            <message>
              <![CDATA[expected:<[Wrong]> but was:<[item.executed === true]>]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.junit.ComparisonFailure: expected:<[Wrong]> but was:<[item.executed === true]>
	at org.junit.Assert.assertEquals(Assert.java:115)
	at org.junit.Assert.assertEquals(Assert.java:144)
	at com.elm.mosa.portal.test.automation.Execution.Data_Consolidation_Dependent(Execution.java:195)
	at com.elm.mosa.portal.test.automation.Configure_Execute.ExecuteDCD(Configure_Execute.java:143)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1129)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:746)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1264)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1189)
	at org.testng.TestNG.runSuites(TestNG.java:1104)
	at org.testng.TestNG.run(TestNG.java:1076)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:152)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:57)
]]>
            </full-stacktrace>
          </exception> <!-- org.junit.ComparisonFailure -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ExecuteDCD -->
        <test-method status="PASS" signature="End()[pri:0, instance:com.elm.mosa.portal.test.automation.Configure_Execute@91151ba]" name="End" is-config="true" duration-ms="780" started-at="2017-02-07T16:31:53Z" finished-at="2017-02-07T16:31:54Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- End -->
        <test-method status="PASS" signature="setup()[pri:0, instance:com.elm.mosa.portal.test.automation.Configure_Execute@91151ba]" name="setup" is-config="true" duration-ms="3541" started-at="2017-02-07T16:31:54Z" finished-at="2017-02-07T16:31:57Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="FAIL" signature="ExecuteEWSD()[pri:19, instance:com.elm.mosa.portal.test.automation.Configure_Execute@91151ba]" name="ExecuteEWSD" duration-ms="10722" started-at="2017-02-07T16:31:57Z" finished-at="2017-02-07T16:32:08Z">
          <exception class="org.junit.ComparisonFailure">
            <message>
              <![CDATA[expected:<[Wrong]> but was:<[item.executed === true]>]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.junit.ComparisonFailure: expected:<[Wrong]> but was:<[item.executed === true]>
	at org.junit.Assert.assertEquals(Assert.java:115)
	at org.junit.Assert.assertEquals(Assert.java:144)
	at com.elm.mosa.portal.test.automation.Execution.Eligibility_WorkSpace_Dependent(Execution.java:201)
	at com.elm.mosa.portal.test.automation.Configure_Execute.ExecuteEWSD(Configure_Execute.java:148)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1129)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:746)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1264)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1189)
	at org.testng.TestNG.runSuites(TestNG.java:1104)
	at org.testng.TestNG.run(TestNG.java:1076)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:152)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:57)
]]>
            </full-stacktrace>
          </exception> <!-- org.junit.ComparisonFailure -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ExecuteEWSD -->
        <test-method status="PASS" signature="End()[pri:0, instance:com.elm.mosa.portal.test.automation.Configure_Execute@91151ba]" name="End" is-config="true" duration-ms="766" started-at="2017-02-07T16:32:08Z" finished-at="2017-02-07T16:32:09Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- End -->
        <test-method status="PASS" signature="setup()[pri:0, instance:com.elm.mosa.portal.test.automation.Configure_Execute@91151ba]" name="setup" is-config="true" duration-ms="4041" started-at="2017-02-07T16:32:09Z" finished-at="2017-02-07T16:32:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="FAIL" signature="ExecuteCESD()[pri:20, instance:com.elm.mosa.portal.test.automation.Configure_Execute@91151ba]" name="ExecuteCESD" duration-ms="10771" started-at="2017-02-07T16:32:13Z" finished-at="2017-02-07T16:32:24Z">
          <exception class="org.junit.ComparisonFailure">
            <message>
              <![CDATA[expected:<[Wrong]> but was:<[item.executed === true]>]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.junit.ComparisonFailure: expected:<[Wrong]> but was:<[item.executed === true]>
	at org.junit.Assert.assertEquals(Assert.java:115)
	at org.junit.Assert.assertEquals(Assert.java:144)
	at com.elm.mosa.portal.test.automation.Execution.Core_Eligibility_Simulation_Dependent(Execution.java:207)
	at com.elm.mosa.portal.test.automation.Configure_Execute.ExecuteCESD(Configure_Execute.java:153)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1129)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:746)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1264)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1189)
	at org.testng.TestNG.runSuites(TestNG.java:1104)
	at org.testng.TestNG.run(TestNG.java:1076)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:152)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:57)
]]>
            </full-stacktrace>
          </exception> <!-- org.junit.ComparisonFailure -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ExecuteCESD -->
        <test-method status="PASS" signature="End()[pri:0, instance:com.elm.mosa.portal.test.automation.Configure_Execute@91151ba]" name="End" is-config="true" duration-ms="795" started-at="2017-02-07T16:32:24Z" finished-at="2017-02-07T16:32:24Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- End -->
        <test-method status="PASS" signature="setup()[pri:0, instance:com.elm.mosa.portal.test.automation.Configure_Execute@91151ba]" name="setup" is-config="true" duration-ms="3953" started-at="2017-02-07T16:32:24Z" finished-at="2017-02-07T16:32:28Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="FAIL" signature="ExecueCED()[pri:21, instance:com.elm.mosa.portal.test.automation.Configure_Execute@91151ba]" name="ExecueCED" duration-ms="10738" started-at="2017-02-07T16:32:28Z" finished-at="2017-02-07T16:32:39Z">
          <exception class="org.junit.ComparisonFailure">
            <message>
              <![CDATA[expected:<[Wrong]> but was:<[item.executed === true]>]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.junit.ComparisonFailure: expected:<[Wrong]> but was:<[item.executed === true]>
	at org.junit.Assert.assertEquals(Assert.java:115)
	at org.junit.Assert.assertEquals(Assert.java:144)
	at com.elm.mosa.portal.test.automation.Execution.Core_Eligibility_Dependent(Execution.java:213)
	at com.elm.mosa.portal.test.automation.Configure_Execute.ExecueCED(Configure_Execute.java:158)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1129)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:746)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1264)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1189)
	at org.testng.TestNG.runSuites(TestNG.java:1104)
	at org.testng.TestNG.run(TestNG.java:1076)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:152)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:57)
]]>
            </full-stacktrace>
          </exception> <!-- org.junit.ComparisonFailure -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ExecueCED -->
        <test-method status="PASS" signature="End()[pri:0, instance:com.elm.mosa.portal.test.automation.Configure_Execute@91151ba]" name="End" is-config="true" duration-ms="752" started-at="2017-02-07T16:32:39Z" finished-at="2017-02-07T16:32:40Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- End -->
        <test-method status="PASS" signature="setup()[pri:0, instance:com.elm.mosa.portal.test.automation.Configure_Execute@91151ba]" name="setup" is-config="true" duration-ms="3535" started-at="2017-02-07T16:32:40Z" finished-at="2017-02-07T16:32:43Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="FAIL" signature="ExecuteDQ()[pri:22, instance:com.elm.mosa.portal.test.automation.Configure_Execute@91151ba]" name="ExecuteDQ" duration-ms="10829" started-at="2017-02-07T16:32:43Z" finished-at="2017-02-07T16:32:54Z">
          <exception class="org.openqa.selenium.NoSuchElementException">
            <message>
              <![CDATA[no such element: Unable to locate element: {"method":"css selector","selector":"tr.ng-pristine:nth-child(9) > td:nth-child(3) > center:nth-child(1) > span:nth-child(1) > img:nth-child(1)"}
  (Session info: chrome=51.0.2704.106)
  (Driver info: chromedriver=2.21.371459 (36d3d07f660ff2bc1bf28a75d1cdabed0983e7c4),platform=Windows NT 6.1 SP1 x86_64) (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 55 milliseconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.53.1', revision: 'a36b8b1cd5757287168e54b817830adce9b0158d', time: '2016-06-30 19:26:09'
System info: host: 'PC0668', ip: '10.31.42.65', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.7.0_71'
*** Element info: {Using=css selector, value=tr.ng-pristine:nth-child(9) > td:nth-child(3) > center:nth-child(1) > span:nth-child(1) > img:nth-child(1)}
Session ID: 2345e76392a583f5b41d6fe9f2b5be89
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities [{platform=XP, acceptSslCerts=true, javascriptEnabled=true, browserName=chrome, chrome={userDataDir=C:\Users\MALKAM~1\AppData\Local\Temp\scoped_dir18928_8451, chromedriverVersion=2.21.371459 (36d3d07f660ff2bc1bf28a75d1cdabed0983e7c4)}, rotatable=false, locationContextEnabled=true, mobileEmulationEnabled=false, version=51.0.2704.106, takesHeapSnapshot=true, cssSelectorsEnabled=true, databaseEnabled=false, handlesAlerts=true, browserConnectionEnabled=false, webStorageEnabled=true, nativeEvents=true, hasTouchScreen=false, applicationCacheEnabled=false, takesScreenshot=true}]]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.NoSuchElementException: no such element: Unable to locate element: {"method":"css selector","selector":"tr.ng-pristine:nth-child(9) > td:nth-child(3) > center:nth-child(1) > span:nth-child(1) > img:nth-child(1)"}
  (Session info: chrome=51.0.2704.106)
  (Driver info: chromedriver=2.21.371459 (36d3d07f660ff2bc1bf28a75d1cdabed0983e7c4),platform=Windows NT 6.1 SP1 x86_64) (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 55 milliseconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.53.1', revision: 'a36b8b1cd5757287168e54b817830adce9b0158d', time: '2016-06-30 19:26:09'
System info: host: 'PC0668', ip: '10.31.42.65', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.7.0_71'
*** Element info: {Using=css selector, value=tr.ng-pristine:nth-child(9) > td:nth-child(3) > center:nth-child(1) > span:nth-child(1) > img:nth-child(1)}
Session ID: 2345e76392a583f5b41d6fe9f2b5be89
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities [{platform=XP, acceptSslCerts=true, javascriptEnabled=true, browserName=chrome, chrome={userDataDir=C:\Users\MALKAM~1\AppData\Local\Temp\scoped_dir18928_8451, chromedriverVersion=2.21.371459 (36d3d07f660ff2bc1bf28a75d1cdabed0983e7c4)}, rotatable=false, locationContextEnabled=true, mobileEmulationEnabled=false, version=51.0.2704.106, takesHeapSnapshot=true, cssSelectorsEnabled=true, databaseEnabled=false, handlesAlerts=true, browserConnectionEnabled=false, webStorageEnabled=true, nativeEvents=true, hasTouchScreen=false, applicationCacheEnabled=false, takesScreenshot=true}]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:206)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:158)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:678)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:363)
	at org.openqa.selenium.remote.RemoteWebDriver.findElementByCssSelector(RemoteWebDriver.java:492)
	at org.openqa.selenium.By$ByCssSelector.findElement(By.java:430)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:355)
	at org.openqa.selenium.support.pagefactory.DefaultElementLocator.findElement(DefaultElementLocator.java:69)
	at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:38)
	at com.sun.proxy.$Proxy7.getAttribute(Unknown Source)
	at pages.DSSProgram_Execution_page.Return_Data_Quality_Result(DSSProgram_Execution_page.java:592)
	at com.elm.mosa.portal.test.automation.Execution.Data_Quality(Execution.java:219)
	at com.elm.mosa.portal.test.automation.Configure_Execute.ExecuteDQ(Configure_Execute.java:163)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1129)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:746)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1264)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1189)
	at org.testng.TestNG.runSuites(TestNG.java:1104)
	at org.testng.TestNG.run(TestNG.java:1076)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:152)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:57)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.NoSuchElementException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ExecuteDQ -->
        <test-method status="PASS" signature="End()[pri:0, instance:com.elm.mosa.portal.test.automation.Configure_Execute@91151ba]" name="End" is-config="true" duration-ms="745" started-at="2017-02-07T16:32:54Z" finished-at="2017-02-07T16:32:55Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- End -->
        <test-method status="PASS" signature="setup()[pri:0, instance:com.elm.mosa.portal.test.automation.Configure_Execute@91151ba]" name="setup" is-config="true" duration-ms="3716" started-at="2017-02-07T16:32:55Z" finished-at="2017-02-07T16:32:59Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="FAIL" signature="ExecuteDC()[pri:23, instance:com.elm.mosa.portal.test.automation.Configure_Execute@91151ba]" name="ExecuteDC" duration-ms="10765" started-at="2017-02-07T16:32:59Z" finished-at="2017-02-07T16:33:09Z">
          <exception class="org.junit.ComparisonFailure">
            <message>
              <![CDATA[expected:<[Wrong]> but was:<[item.executed === true]>]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.junit.ComparisonFailure: expected:<[Wrong]> but was:<[item.executed === true]>
	at org.junit.Assert.assertEquals(Assert.java:115)
	at org.junit.Assert.assertEquals(Assert.java:144)
	at com.elm.mosa.portal.test.automation.Execution.Data_Consolidation(Execution.java:225)
	at com.elm.mosa.portal.test.automation.Configure_Execute.ExecuteDC(Configure_Execute.java:168)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1129)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:746)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1264)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1189)
	at org.testng.TestNG.runSuites(TestNG.java:1104)
	at org.testng.TestNG.run(TestNG.java:1076)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:152)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:57)
]]>
            </full-stacktrace>
          </exception> <!-- org.junit.ComparisonFailure -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ExecuteDC -->
        <test-method status="PASS" signature="End()[pri:0, instance:com.elm.mosa.portal.test.automation.Configure_Execute@91151ba]" name="End" is-config="true" duration-ms="843" started-at="2017-02-07T16:33:09Z" finished-at="2017-02-07T16:33:10Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- End -->
        <test-method status="PASS" signature="setup()[pri:0, instance:com.elm.mosa.portal.test.automation.Configure_Execute@91151ba]" name="setup" is-config="true" duration-ms="3963" started-at="2017-02-07T16:33:10Z" finished-at="2017-02-07T16:33:14Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="FAIL" signature="ExecuteEWS()[pri:24, instance:com.elm.mosa.portal.test.automation.Configure_Execute@91151ba]" name="ExecuteEWS" duration-ms="10858" started-at="2017-02-07T16:33:14Z" finished-at="2017-02-07T16:33:25Z">
          <exception class="org.junit.ComparisonFailure">
            <message>
              <![CDATA[expected:<[Wrong]> but was:<[item.executed === true]>]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.junit.ComparisonFailure: expected:<[Wrong]> but was:<[item.executed === true]>
	at org.junit.Assert.assertEquals(Assert.java:115)
	at org.junit.Assert.assertEquals(Assert.java:144)
	at com.elm.mosa.portal.test.automation.Execution.Eligibility_WorkSpace(Execution.java:232)
	at com.elm.mosa.portal.test.automation.Configure_Execute.ExecuteEWS(Configure_Execute.java:173)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1129)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:746)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1264)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1189)
	at org.testng.TestNG.runSuites(TestNG.java:1104)
	at org.testng.TestNG.run(TestNG.java:1076)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:152)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:57)
]]>
            </full-stacktrace>
          </exception> <!-- org.junit.ComparisonFailure -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ExecuteEWS -->
        <test-method status="PASS" signature="End()[pri:0, instance:com.elm.mosa.portal.test.automation.Configure_Execute@91151ba]" name="End" is-config="true" duration-ms="764" started-at="2017-02-07T16:33:25Z" finished-at="2017-02-07T16:33:26Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- End -->
        <test-method status="PASS" signature="setup()[pri:0, instance:com.elm.mosa.portal.test.automation.Configure_Execute@91151ba]" name="setup" is-config="true" duration-ms="3411" started-at="2017-02-07T16:33:26Z" finished-at="2017-02-07T16:33:29Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="FAIL" signature="ExecuteCESA()[pri:25, instance:com.elm.mosa.portal.test.automation.Configure_Execute@91151ba]" name="ExecuteCESA" duration-ms="10750" started-at="2017-02-07T16:33:29Z" finished-at="2017-02-07T16:33:40Z">
          <exception class="org.junit.ComparisonFailure">
            <message>
              <![CDATA[expected:<[Wrong]> but was:<[item.executed === true]>]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.junit.ComparisonFailure: expected:<[Wrong]> but was:<[item.executed === true]>
	at org.junit.Assert.assertEquals(Assert.java:115)
	at org.junit.Assert.assertEquals(Assert.java:144)
	at com.elm.mosa.portal.test.automation.Execution.Core_Eligibility_Simulation_Applicant(Execution.java:238)
	at com.elm.mosa.portal.test.automation.Configure_Execute.ExecuteCESA(Configure_Execute.java:178)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1129)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:746)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1264)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1189)
	at org.testng.TestNG.runSuites(TestNG.java:1104)
	at org.testng.TestNG.run(TestNG.java:1076)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:152)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:57)
]]>
            </full-stacktrace>
          </exception> <!-- org.junit.ComparisonFailure -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ExecuteCESA -->
        <test-method status="PASS" signature="End()[pri:0, instance:com.elm.mosa.portal.test.automation.Configure_Execute@91151ba]" name="End" is-config="true" duration-ms="780" started-at="2017-02-07T16:33:40Z" finished-at="2017-02-07T16:33:41Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- End -->
        <test-method status="PASS" signature="setup()[pri:0, instance:com.elm.mosa.portal.test.automation.Configure_Execute@91151ba]" name="setup" is-config="true" duration-ms="3991" started-at="2017-02-07T16:33:41Z" finished-at="2017-02-07T16:33:45Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="FAIL" signature="ExecuteCEA()[pri:26, instance:com.elm.mosa.portal.test.automation.Configure_Execute@91151ba]" name="ExecuteCEA" duration-ms="10783" started-at="2017-02-07T16:33:45Z" finished-at="2017-02-07T16:33:56Z">
          <exception class="org.junit.ComparisonFailure">
            <message>
              <![CDATA[expected:<[Wrong]> but was:<[item.executed === true]>]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.junit.ComparisonFailure: expected:<[Wrong]> but was:<[item.executed === true]>
	at org.junit.Assert.assertEquals(Assert.java:115)
	at org.junit.Assert.assertEquals(Assert.java:144)
	at com.elm.mosa.portal.test.automation.Execution.Core_Eligibility_Applicant(Execution.java:244)
	at com.elm.mosa.portal.test.automation.Configure_Execute.ExecuteCEA(Configure_Execute.java:183)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1129)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:746)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1264)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1189)
	at org.testng.TestNG.runSuites(TestNG.java:1104)
	at org.testng.TestNG.run(TestNG.java:1076)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:152)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:57)
]]>
            </full-stacktrace>
          </exception> <!-- org.junit.ComparisonFailure -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ExecuteCEA -->
        <test-method status="PASS" signature="End()[pri:0, instance:com.elm.mosa.portal.test.automation.Configure_Execute@91151ba]" name="End" is-config="true" duration-ms="780" started-at="2017-02-07T16:33:56Z" finished-at="2017-02-07T16:33:56Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- End -->
      </class> <!-- com.elm.mosa.portal.test.automation.Configure_Execute -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
